### PR TITLE
[TS]LPS-171572 Add src attribute later to trigger "load" event.

### PR DESCRIPTION
**Pull request by ces-binhtran**
Hi team,
Please help to review this PR that was implemented by @tototrinh
Thank you

**Reproduce step:** 
https://issues.liferay.com/browse/LPS-171572
**Solution notes**

- This issue doesn't occur in firefox.
- I did some research and found that the iframe load event not firing in Safari/Chrome because iframes generally start with `src="about:blank"` even if the src is specified directly in the HTML or before an iframe node is added to the DOM. And Chrome will wait until the content itself (via the attr method call) is specified to initiate the load event.
- My solution is to set the src Attribute to make it trigger the `load` event.

### CODE REVIEW

**Comment by deborasoliveira** [comment]
ci:test:relevant

**Comment by deborasoliveira** [comment]
I'm taking a look. 👍🏽

**Comment by deborasoliveira** [comment]
ci:test:relevant

**Comment by deborasoliveira** [comment]
Hello!
As the PR has passed the relevant, it is no longer necessary for me to review it.

File: modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded.jsp [hunk 1]
```diff
@@ -16,15 +16,13 @@
 
 <%@ include file="/layout/init.jsp" %>
 
-<%@ include file="/layout/view/embedded_js.jspf" %>
+<%
+UnicodeProperties typeSettingsUnicodeProperties = layout.getTypeSettingsProperties();
+%>
```
Comment (bryceosterhaus) [inline modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded.jsp:21]: This should probably be in the `embedded_js.jspf` file

File: modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf [hunk 2]
```diff
@@ -18,6 +18,11 @@
 	var iframe = A.one('#embeddedIframe');
 
 	if (iframe) {
+		iframe.set(
+			'src',
+			'<%= HtmlUtil.escapeHREF(typeSettingsUnicodeProperties.getProperty("embeddedLayoutURL")) %>'
+		);
```
Comment (bryceosterhaus) [inline modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf:24]: You might want to move this below the `on('load',` part to ensure that the element has the callback before setting the attribute

**Comment by tototrinh** [comment]
Thanks @bryceosterhaus for your feedback. I updated the code, please help take a look again. Thanks.

**Comment by tototrinh** [comment]
Hi @bryceosterhaus, this PR is related to a customer issue and there are no updates for a week, could you please take the time to review this? Thanks.

**Comment by bryceosterhaus** [comment]
Sorry for the delay! I thought I had approved/forwarded it last week but looks like I didn't.

**Comment by bryceosterhaus** [comment]
ci:forward

**Comment by mateusralv** [comment]
Hi @ces-binhtran The failures in the relevant are not related to this PR.

**Comment by tototrinh** [comment]
Thanks @mateusralv for helping check it. @bryceosterhaus could you take a look again? Thanks.

**Comment by bryceosterhaus** [comment]
@tototrinh feel free to manually forward since both dev review is good and CI has only unrelated failures.

If you need me to manually forward, just let me know

**Comment by tototrinh** [comment]
Hi @bryceosterhaus, I have manually forwarded this. I'll go ahead and close this PR. Thanks for your support.

**Comment by bryceosterhaus** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/128658

**Pull request closed without merging**